app/views/shared/_nav_bar.html.erb

```diff
@@ -42,8 +42,7 @@
         <% if current_user %>
           <%= menu_item "List a Ticket", new_ticket_path %>
         <% else %>
-          <% session[:forward_to] = new_ticket_path %>
-          <%= menu_item "List a Ticket", login_path %>
+          <%= menu_item "List a Ticket", login_path(forward_to: new_ticket_path) %>
```

Comment:
sweet! I like this! I think I can use this for the checkout link as well! :star: